lib/ClangImporter/ImportType.cpp

```diff
@@ -806,16 +806,20 @@ namespace {
     }
 
     ImportResult VisitEnumType(const clang::EnumType *type) {
-      auto clangDecl = type->getDecl();
+      auto clangDecl = type->getDecl()->getDefinition();
+      if (!clangDecl) {
+        // FIXME: If the enum has a fixed underlying type, can we use that
+        // instead? Or import it opaquely somehow?
```

Comment:
But the comment was supposed to apply to enums that *do* have the fixed underlying type, and that means we'd be able to import them as raw-valued structs or caseless `@objc` enums—at least in theory. It is very much like the forward-declared struct problem, though.